examples/emberjs/todomvc/app/components/footer.gjs

```diff
@@ -0,0 +1,37 @@
+import { on } from '@ember/modifier';
+
+import { service } from 'ember-primitives';
+
+import Filters from './filters';
+
+function itemLabel(count) {
+  if (count === 0 || count > 1) {
+    return 'items';
+  }
+
+  return 'item';
+}
+
+<template>
+  {{#let (service 'repo') as |repo|}}
+    <footer class="footer">
+      <span class="todo-count">
+        <strong>{{repo.remaining.length}}</strong>
+       {{itemLabel repo.remaining.length}} left
+      </span>
+
+      <Filters />
+
+      {{#if repo.completed.length}}
+        <button class="clear-completed" type="button" {{on "click" repo.clearCompleted}}>
+          Clear completed
+        </button>
+      {{/if}}
+    </footer>
+
+  {{!-- 
+    Looks like the latest todoMVC styles are broken?
```

Comment:
nit: what's broken? I think as long as the app looks consistent with the other apps it should be ok.

Reply:
idk, I'll remove this comment -- it's been too long